macro/pluverse-preamble.tex

```diff
@@ -24,7 +24,7 @@
   language=C,
   tabsize=1,
   %line numbers....
-  escapeinside={@@},
+  escapeinside={(*@}{@*)},
```

Comment:
This one is not easy to write. Any shorter one?

Reply:
I don't know.

Reply:
is it possible to create macros for (*@) and (@*)?